### DISCUSSION TITLE
How dependencies are supposed to be managed after build-reqs is deprecated?

### DISCUSSION THREAD

**Discussion by jjaakko** [kedro-org/kedro#2069] [Idea]
[The documentation ](https://kedro.readthedocs.io/en/stable/kedro_project_setup/dependencies.html) still instructs to use `kedro build-reqs` but when running `kedro build-reqs` I get the following deprecation warning:

`DeprecationWarning: Command 'kedro build-reqs' is deprecated and will not be available from Kedro 0.19.0.`

How should project dependencies be managed after `build-reqs` is not available anymore? Can the documentation be updated accordingly?

**Comment by stichbury** [comment]
Hi @jjaakko -- please can you raise your question over on our Slack organisation (https://slack.kedro.org) if you've not already done so. That's the place for questions about Kedro now and you'll get a response much more swiftly.
